db/db.go

```diff
@@ -22,14 +22,17 @@ func KMToDBPath(dir string, km *km.KeyManager) (string, error) {
 }
 
 // open a database
-func Open(path string, mode os.FileMode, options *bolt.Options) (*bolt.DB, error) {
+func Open(path, userPath string, mode os.FileMode, options *bolt.Options) (*bolt.DB, error) {
```

Comment:
Yes, that's the user's home dir on the device while the app is installed

Reply:
so that is `path` then? The migration is supposed to write it's migration backup files into the tmp dir of the device, as we discussed yesterday. The reason for that is, that if something goes wrong and we do a wrong clean up, the data is erased on the next system start.

Reply:
The problem we have is with Android is that new os versions don’t let apps to write there (on the tempdir) for security reasons, so you need to have your phone rooted to be able to do that. Since not so many users will have it like that, I moved it to a fixed path, safe to write no matter what Android version are you running. On iOS that is not a problem, since they build a safer os